package.json

```
@@ -1,34 +1,25 @@
 {
   "name": "node-fetch",
-  "version": "3.0.0-beta.9",
+  "version": "3.0.0-beta.10",
   "description": "A light-weight module that brings Fetch API to node.js",
-  "main": "./dist/index.cjs",
-  "module": "./src/index.js",
+  "main": "./src/index.js",
   "sideEffects": false,
   "type": "module",
-  "exports": {
-    ".": {
-      "import": "./src/index.js",
-      "require": "./dist/index.cjs"
-    },
-    "./package.json": "./package.json"
-  },
   "files": [
     "src",
     "dist",
     "@types/index.d.ts"
   ],
   "types": "./@types/index.d.ts",
   "engines": {
-    "node": "^10.17 || >=12.3"
+    "node": ">=12.8"
```

What do you think of the following specifier to not include any versions of Node that doesn't full support ESM?

```suggestion
    "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
```

more information in this discussion: https://github.com/sindresorhus/meta/discussions/15#discussioncomment-688195